modules/porous_flow/doc/content/modules/porous_flow/sqa/porous_flow_srs.md

```diff
@@ -1 +1,33 @@
 !template load file=sqa/module_srs.md.template category=porous_flow module=Porous Flow
+
+!template! item key=system-scope
+!! system-scope-begin
+The Porous Flow module is a library of physics for fluid and heat flow in porous media. It is formulated
+in an extremely general manner, so is capable of solving problems with an arbitrary number of phases
+(gas, liquid, etc) and fluid components (species present in each fluid phase), using any set of primary
+variables.
+
+By simply adding pieces of physics together in an input file, the Porous Flow module enables the user
+to model problems with any combination of fluid, heat and geomechanics.
```

Comment:
This is the spot to describe the scope of simulations performed by the module, and is shared between a few documents. Please double-check my work and add more content as desired.

Reply:
```suggestion
By simply adding pieces of physics together in an input file, the Porous Flow module enables the user
to model problems with any combination of fluid, heat, geomechanics and geochemistry. 
```